packages/compiler-sfc/src/parse.ts

```diff
@@ -7,6 +7,8 @@ import {
   SourceLocation
 } from '@vue/compiler-core'
 import { RawSourceMap } from 'source-map'
+import { warn } from '@vue/runtime-core'
```

Comment:
This is a Node.js only compiler so it should not be using the `warn` method from `runtime-core`. Just `console.warn()` is fine for now.